src/components/App.js

```diff
@@ -35,4 +34,33 @@ export default function App() {
   )
 }
 
-// add license(s) per npm
\ No newline at end of file
+// add license(s) per npm
+// design thinking (Not a linear process (think linked list))
+
+// Empathize > Define > Ideate > Prototype > Test
+
+// upgraded "ini" in package-lock.json to 1.3.6 from 1.3.5
+//react router netlify 404: https://dev.to/rajeshroyal/page-not-found-error-on-netlify-reactjs-react-router-solved-43oa
+// for environment variables deployed on netlify, go to Settings > Build & Deploy > Environment > Edit Variables
+
+// node-sass
+// react-fontawesome
+// react-router-dom
+// emailjs   //catch and modal for email successful submission
+// framer-motion
+// add license(s) per pack
+
+//   <header className="App-header">
+//     <img src={logo} className="App-logo" alt="logo" />
+//     <p>
+//       Edit <code>src/App.js</code> and save to reload.
+//     </p>
+//     <a
+//       className="App-link"
+//       href="https://reactjs.org"
+//       target="_blank"
+//       rel="noopener noreferrer"
+//     >
+//       Learn React
+//     </a>
+//   </header>
```

Comment:
@errolgh I'm not sure where these comments came from. They showed up when fixing merge conflicts. If they're not notes to yourself we can remove.